src/main/java/org/folio/services/lines/PoLinesService.java

```diff
@@ -123,20 +125,23 @@ public Future<Void> updatePoLineWithTitle(Conn conn, String id, PoLine poLine, R
   }
 
   public Future<PoLine> createTitle(Conn conn, PoLine poLine) {
-    if (poLine.getPackagePoLineId() != null) {
-      Promise<PoLine> promise = Promise.promise();
-      getPoLineById(conn, poLine.getPackagePoLineId())
-        .onComplete(ar -> {
-          if (ar.failed() || ar.result() == null) {
-            log.error("Can't find poLine with id={}", poLine.getPackagePoLineId());
-            promise.fail(new HttpException(Response.Status.BAD_REQUEST.getStatusCode()));
-          } else {
-            populateTitleForPackagePoLineAndSave(conn, promise, poLine, ar.result());
-          }
-        });
-      return promise.future();
-    }
-    return createTitleAndSave(conn, poLine);
+    return conn.getById(PURCHASE_ORDER_TABLE, poLine.getPurchaseOrderId(), PurchaseOrder.class)
+      .compose(purchaseOrder -> {
+        if (poLine.getPackagePoLineId() != null) {
```

Comment:
Can we invert the `if` condition so that most part will fall on the method indent level?

Reply:
Inverted